portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -109,7 +95,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     long timestamp = System.currentTimeMillis();
     // Get the message input from the form.
     String message = getParameter(request, "message", "");
-
+    
     Entity formSubmissionEntity = new Entity("FormSubmissions");
```

Comment:
In a design perspective, when should we post comments on the website. What if someone doesn't actually enter anything, but press the submit button? What if they don't write a message?

Reply:
+1. Validate input on client side.